modules/programs/autorandr.nix

```diff
@@ -114,6 +114,54 @@ let
           for the documentation of the transform matrix.
         '';
       };
+
+      dpi = mkOption {
+        type = types.nullOr types.int;
+        description = "Output DPI configuration.";
+        default = null;
+        example = 96;
+      };
+
+      scale = mkOption {
+        type = types.nullOr (types.submodule {
+          options = {
+            method = mkOption {
+              type = types.enum ["factor" "pixel" ];
+              description = "Output scaling method.";
+              default = "factor";
+              example = "pixel";
+            };
+
+            x = mkOption {
+              type = types.either types.float types.int;
+              description = "Horizontal scaling factor/pixels.";
+            };
+            y = mkOption {
+              type = types.either types.float types.int;
+              description = "Vertical scaling factor/pixels.";
+            };
+          };
+        });
+        description = ''
+          Output scale configuration.
+
```

Comment:
If you want a new paragraph you unfortunately have to put `</para><para>` instead of an empty line (empty lines are simply ignored).